### PR TITLE
Update staff-onboarding-guide/README.md

### PR DESCRIPTION
**Pull request by natalie-hub**
Updated onboarding with the gcal link and instruction from Google's help site. @justinegeffen - do I need to reference Google's help site?

### CODE REVIEW

File: contributors/onboarding/staff-onboarding-guide/README.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Welcome to Mattermost! This is a general overview of activities that all staff s
 * Review the expense policy: [How to send company money](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money).
 * Attend first your first [COM](https://handbook.mattermost.com/operations/operations/company-cadence#customer-obsession-meeting-aka-com). Join the [Customer Obsession Meeting channel](https://community.mattermost.com/private-core/channels/cust-obs-meeting) and find past COM meetings in the channel to view at your liesure.
```
Comment (justinegeffen) [inline contributors/onboarding/staff-onboarding-guide/README.md]:
```suggestion
* Attend your first [COM](https://handbook.mattermost.com/operations/operations/company-cadence#customer-obsession-meeting-aka-com). Join the [Customer Obsession Meeting channel](https://community.mattermost.com/private-core/channels/cust-obs-meeting) and find past COM meetings in the channel to view at your leisure.
```

File: contributors/onboarding/staff-onboarding-guide/README.md [hunk 2]
```diff
@@ -70,6 +70,8 @@ Welcome to Mattermost! This is a general overview of activities that all staff s
...
 * Use the [Onboarding](https://community.mattermost.com/private-core/channels/onboarding) channel for any questions.
+* Review the [Time Off Policy](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#communicating-time-off) and add the Time Off calendar to your Mattermost gcal.
```
Comment (justinegeffen) [inline contributors/onboarding/staff-onboarding-guide/README.md]:
```suggestion
* Review the [Time Off Policy](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#communicating-time-off) and add the Time Off calendar to your Mattermost Google Calendar. 
```

File: contributors/onboarding/staff-onboarding-guide/README.md (hunk 1 above)
Comment (natalie-hub) [reply contributors/onboarding/staff-onboarding-guide/README.md]: Ack. thank you Justine 👍

**Review by hannaparks — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Looks good to me, @natalie-hub, those reference links are fine. :)

**Pull request merged**